docs/openapi/api-eservice-operations-interop-v1.yaml

```diff
@@ -392,6 +447,33 @@ components:
         versionNumber:
           title: the e-service's version
           type: integer
+    
+    PollingActiveEserviceContent:
+      type: object
+      properties:
+        id:
+          title: Service id
+          type: string
+        basePath:
+          title: base path
+          type: array
+          items:
+            type: string
+        technology:
+          $ref: '#/components/schemas/EserviceTechnology'
+    
+    PollingActiveEserviceResponse:
```

Comment:
Why not returning the same model as `SearchEserviceContent`?
Maybe renaming it to just `EserviceContent`
Let the requester decide which fields it needs